src/constants.js

```diff
@@ -139,3 +139,16 @@ export const SYSTEM_TABLE_COLUMNS = [
         isShownByDefault: true
     }
 ];
+
+// Reports
+
+export const SYSTEMS_PDF_REPORT_TITLE = 'Insights Resource Optimization Systems Report';
+export const SYSTEMS_REPORT_FILE_NAME = 'resource_optimization--';
+
+export const pdfRowKeys = ['display_name', 'os', 'performance_utilization.cpu', 'performance_utilization.memory', 'performance_utilization.max_io',
```

Comment:
I might have missed out the conversation but want to know if there is any specific reason for adding more columns to CSV & JSON reports. Any opinions on having the same `columns` and `name` as PDF? Just for consistency.

Reply:
Hello @r14chandra I had asked this question in the demo meeting and Pau mentioned that we should follow whatever other services are doing. I checked vulnerability service and they include all the keys from API response. Since our response contains nested objects I created seperate keys array contains all the keys for the data visible on the UI and pdf one is based on the mockup. I hope that answer your question.